### PR TITLE
Migrate cron libs to fix cleanup

### PR DESCRIPTION
**Pull request by hogo6002**

### CODE REVIEW

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -142,6 +142,6 @@ RUN locale-gen en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV PYTHONIOENCODING UTF-8
 
-COPY setup_common.sh setup_clusterfuzz.sh start_clusterfuzz.sh setup_mock_metadata.sh start.sh /data/
+COPY setup_common.sh setup_clusterfuzz.sh start_clusterfuzz.sh setup_mock_metadata.sh start.sh Pipfile Pipfile.lock /data/
```
Comment (hogo6002) [inline docker/base/Dockerfile:145]: Do we need to add pipefiles here to install new packages?

Comment (oliverchang) [reply docker/base/Dockerfile:145]: If we add it to src/Pipfile, we don't need to install it in the base image.

File: src/clusterfuzz/_internal/bot/tasks/cron/libs/request_cache.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Google LLC
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/cron/libs/request_cache.py:1]: This isn't usable outside of a web server context. Let's skip/remove this completely for the new cron.

File: src/clusterfuzz/_internal/bot/tasks/cron/libs/issue_management/issue_tracker_utils.py [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+
+def _get_issue_tracker_project_name(testcase=None):
+  """Return issue tracker project name given a testcase or default."""
+  from clusterfuzz._internal.datastore import data_handler
+  job_type = testcase.job_type if testcase else None
+  return data_handler.get_issue_tracker_name(job_type)
+
+
+@request_cache.wrap(capacity=_ISSUE_TRACKER_CACHE_CAPACITY)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/cron/libs/issue_management/issue_tracker_utils.py:47]: Use memoize.wrap instead for this: https://github.com/google/clusterfuzz/blob/c2de16f9c9ae209a56156f69431e0bc48e608969/src/clusterfuzz/_internal/metrics/fuzzer_stats.py#L361

File: Pipfile [hunk 4]
```diff
@@ -11,16 +11,21 @@ psutil = "==5.7.0"
 
 [dev-packages]
 Fabric = "==1.14.1"
+Flask = "==1.1.2"
```
[the remark this answers is not in the document]
Comment (hogo6002) [reply Pipfile]: As it is imported by request_cache.py, we no longer need it. Removed.

**Review by Alan32Liu — APPROVED** [review]

**Review by oliverchang — APPROVED** [review]

**Pull request merged**